state/runtime/precompiled/bls_agg_sigs_verification.go

```diff
@@ -3,8 +3,8 @@ package precompiled
 import (
 	"errors"
 
+	bls "github.com/0xPolygon/bnsnark1/core"
 	"github.com/0xPolygon/polygon-edge/chain"
-	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
```

Comment:
When I mentioned that we should move bnsnark out in a separate repo, I did not mean to remove `signer` from the repo. Could we keep `signer` and make it so its a wrapper around `bnsnark1` repo?

Reply:
It does not make any sense than. it would be just one file in that case

Reply:
I do not follow. If you were to replace `killic` with `bnsnark1` libraries you would not have to remove the `signer` package.

Reply:
My point is that `polybft` should not be aware that we are using `bsnark1` under the hood. The `signer` was already a good abstraction around `Private/Public Keys` and `Signatures`.

Reply:
I have added wrapper around everything bnsnark1 provided inside `bls.go` file